modules/cache-common/src/main/java/org/opensearch/cache/common/tier/TieredSpilloverCacheSettings.java

```diff
@@ -96,6 +99,7 @@ public class TieredSpilloverCacheSettings {
 
     /**
      * Setting which defines the disk cache size to be used within tiered cache.
+     * Similarly, this setting overrides the size setting from the disk tier implementation.
```

Comment:
nit: `Similarly` is in the context of `onheap` setting. But, when tomorrow someone is reading it independently, probably won't make any sense